deployment/tokens/configure_tokens_for_transfers.go

```diff
@@ -83,11 +85,12 @@ func makeApply(tokenRegistry *TokenAdapterRegistry, mcmsRegistry *changesets.MCM
 				}
 			}
 			configureTokenReport, err := cldf_ops.ExecuteSequence(e.OperationsBundle, adapter.ConfigureTokenForTransfersSequence(), e.BlockChains, ConfigureTokenForTransfersInput{
-				ChainSelector:    token.ChainSelector,
-				TokenPoolAddress: tokenPool.Address,
-				RemoteChains:     remoteChains,
-				ExternalAdmin:    token.ExternalAdmin,
-				RegistryAddress:  registry.Address,
+				ChainSelector:                 token.ChainSelector,
+				TokenPoolAddress:              tokenPool.Address,
+				RemoteChains:                  remoteChains,
+				ExternalAdmin:                 token.ExternalAdmin,
+				RegistryAddress:               registry.Address,
+				CustomBlockConfirmationConfig: token.CustomBlockConfirmationConfig,
```

Comment:
Is this parameter valid for all chains? I think that the interface defined in here should be valid for all chain types

Reply:
Agree. Ad-hoc or chain specific parameters should live in an `ExtraConfigs` structure.